test/e2e/notifier/error/error.go

```diff
@@ -53,6 +63,8 @@ func (c *context) testSKipErrorEvent(t *testing.T) {
 	// Resetting original configuration as per test_config
 	defer delete(utils.AllowedEventKindsMap, utils.EventKind{Resource: "v1/pods", Namespace: "dummy", EventType: "error"})
 	utils.AllowedEventKindsMap[utils.EventKind{Resource: "v1/pods", Namespace: "all", EventType: "error"}] = true
+	utils.AllowedEventKindsMap[utils.EventKind{Resource: "v1/services", Namespace: "all", EventType: "error"}] = true
```

Comment:
We should set the original value instead of `true`. We need to fix this for all the occurrences. 
```
serviceEvent := utils.AllowedEventKindsMap[utils.EventKind{Resource: "v1/services", Namespace: "all", EventType: "error"}]
delete(utils.AllowedEventKindsMap, utils.EventKind{Resource: "v1/services", Namespace: "all", EventType: "error"})
defer func() {
    utils.AllowedEventKindsMap[utils.EventKind{Resource: "v1/services", Namespace: "all", EventType: "error"}] = serviceEvent
}
```

Reply:
@PrasadG193 For all other occurrences, I would file a separate PR.

Reply:
Sure! Thank you